graylog2-web-interface/src/views/components/visualizations/number/NumberVisualization.tsx

```diff
@@ -38,7 +38,7 @@ import type { CurrentViewType } from '../../CustomPropTypes';
 const GridContainer = styled.div`
   display: grid;
   grid-template-columns: 1fr;
-  grid-template-rows: 4fr 1fr;
+  grid-template-rows: 4fr auto;
```

Comment:
We can now use `1fr auto` here. It is not changing the layout it is just slightly easer to understand.

Reply:
You are right! Thanks! :)